src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -23,13 +19,18 @@ import {
   getModalCopyWithHarvardLevel,
 } from 'common/utils/recommend';
 import { mainContent } from 'cms-content/recommendations';
-import { getRecommendationsShareUrl } from 'common/urls';
-import { Region, State, getStateName } from 'common/regions';
-import VaccinationEligibilityBlock from 'components/VaccinationEligibilityBlock';
 import { EventCategory, EventAction, trackEvent } from 'components/Analytics';
+import CompareMain from 'components/Compare/CompareMain';
+import ErrorBoundary from 'components/ErrorBoundary';
+import Explore, { ExploreMetric } from 'components/Explore';
+import Recommend from 'components/Recommend';
+import ShareModelBlock from 'components/ShareBlock/ShareModelBlock';
+import VaccinationEligibilityBlock from 'components/VaccinationEligibilityBlock';
 import VulnerabilitiesBlock from 'components/VulnerabilitiesBlock';
-import { useCcviForFips } from 'common/hooks';
-import { useScrollToElement } from 'common/hooks';
+import ChartBlock from './ChartBlock';
+import LocationPageBlock from './LocationPageBlock';
+import LocationPageHeader from './LocationPageHeader';
+import { ChartContentWrapper } from './ChartsHolder.style';
```

Comment:
I sorted the dependencies a bit and merged two imports from the same file into one.